dirsrvtests/tests/suites/disk_monitoring/disk_monitoring_test.py

```diff
@@ -731,6 +731,38 @@ def test_valid_operations_are_permitted(topo, setup, reset_logs):
     assert topo.standalone.config.set('nsslapd-disk-monitoring-logging-critical', 'on')
 
 
+def test_enable_external_libs_debug_log(topo, setup, reset_logs):
```

Comment:
IIRC this test should have a marker or warning of some kind about how it will use disk space. Also some filesystems will see a dev/zero dd and just make a sparse file and can over-allocate ... there are also risks with corruption on some fs's (btrfs famous enospc issues).

Reply:
Good catch! Fixed:)